rpmalloc/rpmalloc.cpp

```diff
@@ -99,6 +99,13 @@
 #  define assert(x)
 #endif
 
+void Crash()
+{
+		*(int*)(nullptr) = 42;
```

Comment:
Why not take a `const char* message` and use that instead of 42? It will allow to make a more understandable error like "maximum heaps count reached"

Reply:
Done